card.go

```diff
@@ -179,3 +179,39 @@ func (c *Card) AddComment(text string) ([]byte, error) {
 	body, err := c.client.Post("/cards/"+c.Id+"/actions/comments", payload)
 	return body, err
 }
+
+// Archive will archive the card
+// https://developers.trello.com/advanced-reference/card#put-1-cards-card-id-or-shortlink-closed
+func (c *Card) Archive() ([]byte, error) {
+	payload := url.Values{}
+	payload.Set("value", "true")
+
+	return c.client.Put("/cards/"+c.Id+"/closed", payload)
+}
+
+// SendToBoard will dearchive the card, or send the card to the board back from archive
+// https://developers.trello.com/advanced-reference/card#put-1-cards-card-id-or-shortlink-closed
+func (c *Card) SendToBoard() ([]byte, error) {
+	payload := url.Values{}
+	payload.Set("value", "false")
+
+	return c.client.Put("/cards/"+c.Id+"/closed", payload)
+}
+
+// MoveToList will move the card to another list
+// https://developers.trello.com/advanced-reference/card#put-1-cards-card-id-or-shortlink-idlist
+func (c *Card) MoveToList(listId string) ([]byte, error) {
```

Comment:
I'd keep this to one of those methods to avoid confusion. In the end I don't have a preference for one of them…